src/Illuminate/Foundation/Providers/ConsoleSupportServiceProvider.php

```diff
@@ -25,5 +26,6 @@ class ConsoleSupportServiceProvider extends AggregateServiceProvider
         'Illuminate\Database\SeedServiceProvider',
         'Illuminate\Foundation\Providers\ComposerServiceProvider',
         'Illuminate\Queue\ConsoleServiceProvider',
+        JobsServiceProvider::class,
```

Comment:
Is it worth swapping them all to that notation? It works really nice when refactoring apposed to using string class names?
I've seen it in other parts of Laravel that way, is there a preference as to which notation should be used?